static/index.html

```diff
@@ -100,6 +100,16 @@
                     <path d="M382 802 168 588l38-38 176 176 374-374 38 38-412 412Z" />
                   </svg>
                 </button>
+                <button id="invalidateBtn" style="display: none">
+                  <div>Invalidate</div>
```

Comment:
```suggestion
                  <div>Void</div>
```

Because this makes the button jitter due to the length of the word "invalidate"